server/handlers/meshery_filter_handler.go

```diff
@@ -204,7 +204,7 @@ func (h *Handler) handleFilterPOST(
 //
 // ```?pagesize={pagesize}``` Default pagesize is 10
 //
-// ```?visibility={visibility}``` Default visibility is public
+// ```?visibility={visibility}``` Default visibility is private
```

Comment:
@Aritra8438, have u tested it? if the default visibility is set to private, public filters are still showing up
![image](https://github.com/meshery/meshery/assets/82499435/cb34690b-ccc6-4e12-862e-a8155c12c353)
  
And even when set to private, public filters still showing up
  
![image](https://github.com/meshery/meshery/assets/82499435/00ac8d9d-02e2-4260-a7e5-e88bcb85b60a)

Reply:
I think need to update the `mesheryFilterPage` inside ` GetMesheryFilters` to include visibility to make it work

Reply:
Hi, @Freedisch. Yes, I tested that if visibility is set to `public`, then no private filters are showing up.
Can you, suggest me how I add a `public`/`published` filter?

That's the thing I should test in this case I think.

Reply:
```go
query := mfp.DB.Where("visibility = ?", visibility).Order(order)
```
This is where I'm filtering based on filters.

Reply:
I see now, this query doesn'

> Hi, @Freedisch. Yes, I tested that if visibility is set to `public`, then no private filters are showing up. Can you, suggest me how I add a `public`/`published` filter?
> 
> That's the thing I should test in this case I think.

there is only public filter because you are on page 0 where only public filters are presents, try it at page 1 where both public and private filters are present

![image](https://github.com/meshery/meshery/assets/82499435/322fda36-fc2c-491f-a56f-6b5865847a57)

Reply:
btw, u might not see it if u don't have private filter on the playground

Reply:
Is this comment unresolved, @Aritra8438?